.env

```diff
@@ -0,0 +1,10 @@
+POSTGRES_PASSWORD=password
+POSTGRES_USER=postgres
+POSTGRES_DB=postgres
+DATABASE_URL=postgresql://postgres:password@database:5432/postgres
+CACHE_TYPE=ttl
+CACHE_TTL=120
+CACHE_MAXSIZE=1000
+VITE_BACKEND_URL=http://localhost:8000
+REACT_APP_ADMIN_URL=http://localhost:3000
+REACT_APP_ADMIN=true
```

Comment:
So we're going to have just one big .env file? Is that good practice?

Reply:
ehhh with our size it probably doesn't matter but we'll split it